docs/versioning_cli_test.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import pytest
+from s3_helpers import run_example
+from botocore.exceptions import ClientError
+from shlex import split, quote
+import subprocess
+
+config = "../params/br-se1.yaml"
+
+
+pytestmark = [pytest.mark.bucket_versioning, pytest.mark.bucket_versioning_cli]
```

Comment:
será que não seria melhor termos um marker cli apenas e na hora do filter fazer "bucket_versioning and not cli" para excluir o que for cli? nao pensei muito sobre como usaremos estas marcas... mas me parece util ter uma forma de incluir ou excluir testes de cli de varios

Reply:
Concordo, irei trocar a classe versioning_cli para cli, também ja faço isso para o locking